### PR TITLE
Adds new rooms and marks old ones as depricated

### PR DESCRIPTION
**Pull request by I3uckwheat**

### CODE REVIEW

**Comment by I3uckwheat** [comment]
I think i got it all

**Comment by I3uckwheat** [comment]
I wasn't sure about the secret bot room

**Comment by codyloyd** [comment]
you screwed up the quotes and missed a comma or two 👀 
image

**Comment by I3uckwheat** [comment]
Ooops... I will fix

File: config.js [hunk 1]
```diff
@@ -7,22 +7,23 @@ var config = {
...
-      // 'TheOdinProject/Getting-Hired',
-      // 'TheOdinProject/Git',
-      // 'TheOdinProject/HTML-CSS',
-      // 'TheOdinProject/Interesting-Stuff',
-      // 'TheOdinProject/Javascript',
-      // 'TheOdinProject/ModTeam',
-      // 'TheOdinProject/Rails',
-      // 'TheOdinProject/Random',
-      // 'TheOdinProject/Ruby',
+      'TheOdinProject/General',
```
Comment (JonathanYiv) [inline config.js]: This one is still `TheOdinProject/theodinproject`

File: config.js [hunk 2]
```diff
@@ -7,22 +7,23 @@ var config = {
...
+      'TheOdinProject/Contributing',
+      'TheOdinProject/Getting-Hired',
+      'TheOdinProject/ModTeam',
+      
+      //Depricated
+      'TheOdinProject/Random',
+      'TheOdinProject/Javascript',
+      'TheOdinProject/Rails',
+      'TheOdinProject/Ruby',
+      'TheOdinProject/Interesting-Stuff',
```
Comment (JonathanYiv) [inline config.js]: This doesn't exist anymore

File: config.js [hunk 3]
```diff
@@ -7,22 +7,23 @@ var config = {
...
+      'TheOdinProject/vim',
```
Comment (JonathanYiv) [inline config.js]: This doesn't exist anymore

File: config.js [hunk 4]
```diff
@@ -7,22 +7,23 @@ var config = {
...
+      'TheOdinProject/HTML-CSS',
       // 'TheOdinProject/theodinproject',
-      // 'TheOdinProject/LookingForPairs',
-      // 'TheOdinProject/New-JS-course',
       // 'codyloyd',
       'TheOdinProject/secretbotroom'
```
Comment (JonathanYiv) [inline config.js]: This doesn't exist anymore

Comment (codyloyd) [reply config.js]: yes it does.. it's a private room that I use for local dev

Comment (JonathanYiv) [reply config.js]: Oo, didn't know that. My bad

**Pull request merged**